pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -115,5 +119,47 @@ func NewConfigObserver(
 	configInformer.Config().V1().Networks().Informer().AddEventHandler(c.EventHandler())
 	configInformer.Config().V1().Schedulers().Informer().AddEventHandler(c.EventHandler())
 
+	registerMetrics(configInformer)
+
 	return c
 }
+
+// registerMetrics exposes core platform metrics that relate to the configuration
+// of Kubernetes.
+// TODO: in the future this may move to cluster-config-operator.
+func registerMetrics(configInformer configinformers.SharedInformerFactory) {
```

Comment:
since you have two, start us a little config metrics package called from starter.go?  We'll lose track of it here.

Reply:
sure